website/sidebars.js

```diff
@@ -392,11 +392,16 @@ const sidebarSettings = {
             {
               type: "link",
               label: "API v2",
```

Comment:
It might be helpful to point out these are legacy. 
```suggestion
              label: "API v2 (Legacy docs)",
```